lib/src/api/page_loader_element_interface.dart

```diff
@@ -65,17 +65,17 @@ abstract class PageLoaderElement extends PageLoaderSource {
   /// The shadow root hosted by this element.
   PageLoaderElement get shadowRoot;
 
-  /// The text content of this element.
+  /// The inner text of this element.
   ///
-  /// This should return the value of `<element>.innerText` (see
-  /// https://developer.mozilla.org/en-US/docs/Web/API/Node/innerText), but
-  /// behaviour may vary for the different pageloader implementations.
+  /// This should return the value of
+  /// [`<element>.innerText`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText),
```

Comment:
Please fix this to be within the 80-length limit. Markdown should still work even if split:
https://stackoverflow.com/questions/19985235/break-long-lines-in-markdown-code

Reply:
Done.